src/components/vacation-requests-table/vacation-requests-table-toolbar/vacation-requests-table-toolbar.tsx

```diff
@@ -77,6 +77,7 @@ const TableToolbar = ({
    *
    * @param selectedRowIds selected row ids
    */
+  const disableEdit = selectedRowIds?.length === 1 && rows.find((request: VacationsDataGridRow) => request.id === selectedRowIds[0])?.status !== VacationRequestStatuses.PENDING
```

Comment:
```suggestion
  const disableEdit = selectedRowIds?.length === 1 && rows.find((request: VacationsDataGridRow) => request.id === selectedRowIds[0])?.status !== VacationRequestStatuses.PENDING;
  
```

Reply:
The suggestion above is adding missing semi colon and a line break before the function.
You have also added  this const between the function (toggleToolbarOpenOnSelectedRowsIds) and its functions comment/ documentation (TSDocs). You should move the disable edit const to be before the function comment

Reply:
I realise I suggested disableEdit as the name in a previous comment but perhaps isDisabled or disableEditButton is a clearer name than disableEdit.